monai/networks/nets/unet.py

```diff
@@ -57,6 +57,12 @@ def __init__(
             act: activation type and arguments. Defaults to PReLU.
             norm: feature normalization type and arguments. Defaults to instance norm.
             dropout: dropout ratio. Defaults to no dropout.
+
```

Comment:
I think these so depend on the kernel sizes, it's not easy to write down all the viable rules, I'd suggest a tutorial like this https://gist.github.com/fepegar/1fb865494cb44ac043c3189ec415d411

Reply:
Hi @wyli ,

Thanks for sharing the interesting tutorial.
Does our UNet implementation follow the rules as the tutorial? If so, I think we can put the link of this tutorial in the doc-string directly? I saw the tutorial said the implementation of NiftyNet seems easier to write down the rule: "In NiftyNet, the convolutional layers of U-Net use SAME padding. This means that I must just be divisible by 2^N".

Thanks.

Reply:
CC @ericspod for the discussion.
Thanks.

Reply:
@ericspod @rijobro , what do you guys think? Should we put the link of this tutorial in the doc-string directly? Or some pattern to define the input data size?

Thanks.

Reply:
The MONAI unet implementation is more flexible, probably the tutorial doesn't fit here. I feel itd be nice to have a tutorial page to show its typical configurations, parameters, model sizes, memory usage etc. Or simply use the basicunet, the setup is straightforward...

Reply:
OK, I submitted a ticket in the tutorial repo: https://github.com/Project-MONAI/tutorials/issues/280 to track this task.

Thanks.